metagov/metagov/core/models.py

```diff
@@ -139,32 +246,43 @@ def send_event_to_driver(self, event_type: str, data: dict, initiator: dict):
                 f"Error sending event to driver at {settings.DRIVER_EVENT_RECEIVER_URL}: {resp.status_code} {resp.reason}"
             )
 
-    def add_linked_account(self, *, platform_identifier, external_id=None, custom_data=None, link_type=None, link_quality=None):
+    def add_linked_account(
+        self, *, platform_identifier, external_id=None, custom_data=None, link_type=None, link_quality=None
+    ):
         """Given a platform identifier, creates or updates a linked account. Also creates a metagov
         id for the user if no external_id is passed in.
         """
         from metagov.core import identity
 
-        optional_params = {"community_platform_id": self.community_platform_id, "custom_data": custom_data,
-            "link_type": link_type, "link_quality": link_quality}
```

Comment:
fyi I have auto-formatting with [black](https://black.readthedocs.io/en/stable/) set up in my editor @shaunagm that's why there are all these diffs. I didn't make any changes to identity.

Reply:
and if you want to set it up locally then we can have matching code :)

Reply:
you are not the first person to suggest I set up black... maybe someday ;)